R/dplyr_methods.R

```diff
@@ -892,23 +892,21 @@ group_split.SingleCellExperiment <- function(.data, ...) {
   
   var_list <- enquos(...)
   
-  .data <- .data |> 
-      unite.SingleCellExperiment("group_col", !!!var_list, remove = FALSE)
-  
   group_df <- .data |> 
       as_tibble()
   
-  group_list <- group_df$group_col |> 
-      unlist() |> 
+  group_df <- group_df |> 
+      unite("group_col", !!!var_list, remove = FALSE)
```

Comment:
better to use `group_column___` rather than `group_col` so we make almost sure this column does not exist already in someone dataset.